eu-north-1/remote_backend.tf

```diff
@@ -0,0 +1,25 @@
+# Create S3 bucket
+resource "aws_s3_bucket" "terraform_state" {
```

Comment:
You have to create the state file manually, the state file are configured during the initialisation phase.